### PR TITLE
docs: add Secret chinese document to Admin API

### LINKED ISSUE #9520: docs: secrets documentation of admin-api is needed
### Current State

Current the secrets docs of admin-api only have english versioin, we need chinese version docs.

### Desired State

ref: https://apisix.apache.org/docs/apisix/admin-api/#secret

### PR DESCRIPTION
**Pull request by zccabb**
### Description

Provide Chinese documentation for Admin API Secret.

https://apisix.apache.org/docs/apisix/admin-api/#secret

Fixes #9520 

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [x] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

**Comment by monkeyDluffy6017** [comment]
I think you should read what you have translated and compare it with other translations, it could be a bit more better

File: docs/zh/latest/admin-api.md [hunk 1]
```diff
@@ -1394,3 +1394,67 @@ Plugin 资源请求地址：/apisix/admin/stream_routes/{id}
 | protocol.conf    | 否    | 配置     | 协议特定的配置。                                                               |                    |
 
 你可以查看 [Stream Proxy](./stream-proxy.md#更多-route-匹配选项) 了解更多过滤器的信息。
+
+## Secret
+
+安全密钥 (Secret) 表示表示 `密钥管理(Secrets Management)`，可以使用任何受支持的密钥管理器，例如 `vault`。
```
[the remark this answers is not in the document]
Comment (zccabb) [reply docs/zh/latest/admin-api.md]: done

File: docs/zh/latest/admin-api.md [hunk 2]
```diff
@@ -1394,3 +1394,66 @@ Plugin 资源请求地址：/apisix/admin/stream_routes/{id}
...
+```shell
+HTTP/1.1 200 OK
+...
+
+{"key":"\/apisix\/secrets\/vault\/test2","value":{"id":"vault\/test2","token":"apisix","prefix":"apisix","update_time":1669625828,"create_time":1669625828,"uri":"http:\/\/xxx\/get"}}
+```
+
+### 响应参数 {#secret-config-response-parameters}
+
+当前，响应是从 etcd 返回的。
```
Comment (monkeyDluffy6017) [inline docs/zh/latest/admin-api.md:1459]:
```suggestion

## Secret

Secret 指的是 `Secrets Management`（密钥管理），可以使用任何支持的密钥管理器，例如 `vault`。

### 请求地址

Secret 资源请求地址：/apisix/admin/secrets/{secretmanager}/{id}

### 请求方法

| 方法   | 请求URI                                     | 请求主体 | 描述                                                         |
| ------ | ------------------------------------------- | -------- | ------------------------------------------------------------ |
| GET    | /apisix/admin/secrets                       | NULL     | 获取所有 secret 的列表。                                     |
| GET    | /apisix/admin/secrets/{manager}/{id}        | NULL     | 根据 id 获取指定的 secret 。                                 |
| PUT    | /apisix/admin/secrets/{manager}             | {…}      | 创建新的 secret 配置。                                       |
| DELETE | /apisix/admin/secrets/{manager}/{id}        | NULL     | 删除具有指定 id 的 secret。                                  |
| PATCH  | /apisix/admin/secrets/{manager}/{id}        | {…}      | 更新指定 secret 的选定属性。如果要删除一个属性，可以将该属性的值设置为null。 |
| PATCH  | /apisix/admin/secrets/{manager}/{id}/{path} | {…}      | 更新路径中指定的属性。其他属性的值保持不变。                 |

### body 请求参数

当`{secretmanager}`是`vault`时：

| 参数       | 是否必需 | 类型   | 描述               | 示例 |
| ---------- | -------- | ------ | ------------------ | ---- |
| uri        | 是       | URI    | Vault服务器的URI。 |      |
| prefix     | 是       | string | 密钥前缀           |      |
| token      | 是       | string | Vault令牌。        |      |

配置示例： 

```
{        
    "uri": "https://localhost/vault",
    "prefix": "/apisix/kv",
    "token": "343effad"
}
```

使用示例：

```
curl -i http://127.0.0.1:9180/apisix/admin/secrets/vault/test2 \
-H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
{
    "uri": "http://xxx/get",
    "prefix" : "apisix",
    "token" : "apisix"
}'
```

```
HTTP/1.1 200 OK

{"key":"\/apisix\/secrets\/vault\/test2","value":{"id":"vault\/test2","token":"apisix","prefix":"apisix","update_time":1669625828,"create_time":1669625828,"uri":"http:\/\/xxx\/get"}}
```

### 应答参数

当前的响应是从etcd返回的。

```

**Comment by zccabb** [comment]
fixed,Thanks to @monkeyDluffy6017 for all your help

**Review by monkeyDluffy6017 — APPROVED** [review]

**Review by monkeyDluffy6017 — APPROVED** [review]

**Review by SylviaBABY — APPROVED** [review]

**Pull request merged**